docs/fields_specification.md

```diff
@@ -15,27 +15,32 @@ The set of primitive type names is:
 
 So, the previous types can be matched to:
 
-| Avro Type | Python Type |
-|-----------|-------------|
-| string    |     str     |
-| long      |     int     |
-| boolean   |     bool    |
-| double    |     float   |
-| null      |     None    |
-| bytes     |     bytes   |
+| Avro Type    | Python Type |
+|--------------|-------------|
+| string       |     str     |
+| int,long     |     int     |
+| boolean      |     bool    |
+| float,double |     float   |
+| null         |     None    |
+| bytes        |     bytes   |
+
```

Comment:
Good explanation. Maybe we can also add two new rows for the new types in order to make it more clear for the users, what do you think?  

| Avro Type | Python Type |
|-----------|-------------|
| string    |     str     |
| long      |     int     |
| boolean   |     bool    |
| double    |     float   |
| null      |     None    |
| bytes     |     bytes   |
|  float  |    types.Float32   |
|   int   |   types.Int32  |

Reply:
Done.